app/models/efile_submission.rb

```diff
@@ -84,10 +84,14 @@ def self.statefile_state_counts(except: [])
     ActiveRecord::Base.connection.execute(<<~SQL).each { |row| result[row['to_state']] = row['count'] }
       SELECT to_state, COUNT(*) FROM "efile_submissions"
       LEFT OUTER JOIN efile_submission_transitions AS most_recent_efile_submission_transition ON (
-        efile_submissions.id = most_recent_efile_submission_transition.efile_submission_id AND 
+        efile_submissions.id = most_recent_efile_submission_transition.efile_submission_id AND
         most_recent_efile_submission_transition.most_recent = TRUE
       )
       WHERE most_recent_efile_submission_transition.to_state IS NOT NULL
+      AND (
```

Comment:
Not a blocker, but you could make this a little more concise:
``` AND efile_submissions.data_source_type IN ('StateFileAzIntake', 'StateFileNyIntake') ```

Reply:
oh nice thank you @tofarr

Reply:
Hmm I can't get it to work when I try this @tofarr